docs/4-roles-expectations.md

```diff
@@ -14,33 +14,53 @@ It should be noted that the Guild as an entity does not directly manage or parti
 
 ## 4.2 Members
 
-For the pilot, members of the Protocol Guild will have up to three different functions: [slot holders](https://protocol-guild.readthedocs.io/en/latest/4-roles-expectations.html#slot-holders), [curators](https://protocol-guild.readthedocs.io/en/latest/4-roles-expectations.html#curators) and [signers](https://protocol-guild.readthedocs.io/en/latest/4-roles-expectations.html#signers).
+For the Pilot, members of the Protocol Guild will have up to three different functions: [slot holders](https://protocol-guild.readthedocs.io/en/latest/4-roles-expectations.html#slot-holders), [curators](https://protocol-guild.readthedocs.io/en/latest/4-roles-expectations.html#curators) and [signers](https://protocol-guild.readthedocs.io/en/latest/4-roles-expectations.html#signers).
 
 ### 4.21 Slot Holders
 
 Slot holders are members of the Guild who have qualified for a placement (slot) in the split contract. Estimated to be around 150-200 individuals, however the actual number may be higher or lower. There is no cap or target for number of slots. Slots can be set to any Ethereum address, including the individuals own, a charity, or another split contract.
 
 #### Qualifications
 
-Members should be committed to Ethereum and its ethos of decentralization. The contributions of qualifying individuals should be:
+Note 1: these guidelines will change over time, i.e. become more restrictive in some places and more permissive in others.
+Note 2: contributing to the projects/repos referenced below is necessary but does not guarantee Guild eligibility. While this list tries to be explicit by linking to example repos, there are some research areas which can't be linked to a single repo.
```

Comment:
```suggestion

Note 2: contributing to the projects/repos referenced below is necessary but does not guarantee Guild eligibility. While this list tries to be explicit by linking to example repos, there are some research areas which can't be linked to a single repo.
```

Reply:
maybe i'm reading this wrong - are you deleting note 1 or just adding a new line between both of them?